### PR TITLE
feat(storybook): Create stories for Badge

### PR DESCRIPTION
**Pull request by laughnan**
This PR creates Storybook stories for `Badge`.

Screen Shot 2020-01-12 at 16 43 33

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @laughnan has signed the CLA. Thanks for the pull request!

**Comment by laughnan** [comment]
can you give this a look @sahiga ?

File: src/components/badge/__tests__/Badge.stories.test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe('components/badge/Badge', () => {
+    test.each([['components-badge--regular']])('looks visually correct when using story %s', async id => {
```
Comment (sahiga) [inline src/components/badge/__tests__/Badge.stories.test.js]: Can you add tests for the other states?

File: src/components/badge/Badge.stories.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export const warning = () => <Badge type="warning">Warning Badge</Badge>;
+
+export const highlight = () => <Badge type="highlight">Highlight Badge</Badge>;
+
+export const error = () => <Badge type="error">Error Badge</Badge>;
+
+export const alert = () => <Badge type="alert">Alert Badge</Badge>;
+
+export const success = () => <Badge type="success">Success Badge</Badge>;
```
Comment (sahiga) [inline src/components/badge/Badge.stories.js]: Can you add stories for the prebuilt `BETA` and `UPGRADE` badges as well?

Comment (laughnan) [reply src/components/badge/Badge.stories.js]: totally! i remember seeing those and then forgetting to push them to this story. doing now...

**Pull request merged**